src/GeositeFramework/js/PluginBase.js

```diff
@@ -79,6 +80,7 @@ define(["dojo/_base/declare",
             constructor: function(args) {
                 isBlacklisted = _.partial(_.contains, Geosite.app.data.region.identifyBlacklist);
                 declare.safeMixin(this,args);
+                if ($().slider) { pips.addjQueryUISliderPips($); }
```

Comment:
I think this is a leftover from getting the imports set up. Let me drop it in a fixup commit and see if it ever does fail; it shouldn't now.